### ISSUE TITLE
App running locally but fails with "ERROR" message in spaces 

### ISSUE THREAD

**Issue by santarabantoosoo** [gradio-app/gradio#1914]
### Describe the bug

The error message appears only when I click the submit button (only in the middle tab) "word frequency" 

Link for the space 
https://huggingface.co/spaces/Santarabantoosoo/Sentiments_topic_modeling_ITALIAN 

My problem is not related to examples as discussed in previous issues. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

https://huggingface.co/spaces/Santarabantoosoo/Sentiments_topic_modeling_ITALIAN

### Screenshot

![image](https://user-images.githubusercontent.com/31887357/182008631-51eafbf0-785d-4d0e-942b-f7902e897317.png)

![image](https://user-images.githubusercontent.com/31887357/182008647-d525e3b6-b60f-4086-a4fd-35672f7c1602.png)

### Logs

```shell
I have set debug = True. However, I can see nothing except ERROR message in the output area
```

### System Info

```shell
gradio version 3.1.1.
```

### Severity

blocking all usage of gradio

**Comment by freddyaboulton** [comment]
@santarabantoosoo Thank you for filing this issue!

On spaces, there is a button where you can see the log of your space. It's right next to the green "Running" status indicator.

![image](https://user-images.githubusercontent.com/41651716/182188827-68e7803f-908e-4b8d-94f0-7743afd5c237.png)

Can you tell us if there is anything helpful there? Usually exceptions are logged there.

**Comment by santarabantoosoo** [comment]
Thanks a lot. 

I can no longer see the error

I think this is related to fix #1925

**Comment by abidlabs** [comment]
Ok thanks for letting us know @santarabantoosoo!
